WooCommerce/UITestsFoundation/Screens/Orders/OrdersScreen.swift

```diff
@@ -2,10 +2,6 @@ import ScreenObject
 import XCTest
 
 public final class OrdersScreen: ScreenObject {
-
-    // TODO: Remove force `try` once `ScreenObject` migration is completed
-    public let tabBar = try! TabNavComponent()
-
```

Comment:
When we made the test fail on create order/edit order screen, this was causing the test to not fail gracefully (it was in a stuck state), since tab nav is not part of Orders Screen anyways, I'm removing this from here. On create/edit order, the tab nav is not part of the orders screen, the only time it is part of the screen is on orders list.